src/PoolManager.sol

```diff
@@ -399,7 +401,11 @@ contract PoolManager is Auth, IPoolManager {
         address restrictionManager = restrictionManagerFactory.newRestrictionManager(
             undeployedTranche.restrictionSet, token, restrictionManagerWards
         );
+
         TrancheTokenLike(token).file("restrictionManager", restrictionManager);
+        if (cfgRouter != address(0)) {
```

Comment:
With the change I proposed in the router contract, the router just acts as an operator of the investor, and does not even need to be a member

Reply:
So if we do that + move endorsements to the investment manager, I think the pool manager doesn't even need to know about the router anymore

Reply:
Actually that's not true. It won't need to be a member anymore for "normal" requests, but it will need to be a member for locked requests, since then it needs to take custody of the assets